### DISCUSSION TITLE
Bypass "Please verify your age" popup [FEATURE REQUEST]

### DISCUSSION THREAD

**Discussion by Nerxxy** [ReVanced/revanced-discussions#443] [Question] [answered]
A lot of videos that I watch generally aren't age restricted but I get the odd one every now and then on creators that dont post content that needs age restrictions for example: 

Darkviperau:
![Screenshot_20220805-054229.png](https://user-images.githubusercontent.com/37718547/183003102-fad390ca-494d-4eb6-be37-158c17dd9c16.png)

![Screenshot_20220805-054247.png](https://user-images.githubusercontent.com/37718547/183003118-a86770d6-56eb-416d-a5d3-b803bce1abda.png)

The video thats age restricted is no different to the video posted before it... I have already watched it on my pc as I have tampermonkey script that bypasses the restriction. I wanted to watch it again as I was bored and being away from my pc I cant.

**Comment by reisxd** [comment, marked as answer]
Dupe of #118
